### ISSUE TITLE
[a11y tool] indicate edges

### ISSUE THREAD

**Issue by pkra** [zorkow/speech-rule-engine#26]
When walking an expression, the a11y tool should announce that it reached an edge (i.e., the right/left-most node of a level, the root or a leaf).

Regarding the F2F discussion about earcons. From a quick search I found [this](http://stackoverflow.com/questions/17762763/play-wav-sound-file-encoded-in-base64-with-javascript): play a base64-encoded audio file via an audio element (support IE9+)

```javascript
var Sound = (function () {
    var df = document.createDocumentFragment();
    return function Sound(src) {
        var snd = new Audio(src);
        df.appendChild(snd); // keep in fragment until finished playing
        snd.addEventListener('ended', function () {df.removeChild(snd);});
        snd.play();
        return snd;
    }
}());
// then do it
var snd = Sound("data:audio/wav;base64," + base64string);
```

**Comment by zorkow** [comment]
See https://github.com/mathjax/MathJax-RespEq/pull/38

**Comment by pkra** [comment]
Thanks. Which bugs should I file where?

**Comment by zorkow** [comment]
Done  mathjax/MathJax-RespEq#38
